workflow.json

```
@@ -0,0 +1,70 @@
+{
+  "workflow_id": "workflow_com_if_e_switch",
+  "start": "n1",
+  "nodes": {
+    "n1": {
```

Aqui seria interessante retirarmos a letra n relacionada a nomenclatura, porque nós já estamos dentro da referenciação dos nodes. O que acha? Mesma coisa para os outros.
```suggestion
    "1": {
```

---

De acordo, removi o n, e apresentei um layout novo para os próximos nós.